### ISSUE TITLE
DoNotUseDirectReceiverReferenceInsideWith false positives on named parameters

### ISSUE THREAD

**Issue by ursusursus** [Faire/faire-detekt-rules#124]
Hi, I like the idea of `DoNotUseDirectReceiverReferenceInsideWith` but it trips up on named parameters

```
val request = with(entrepreneur) { <---------
    ConfirmCustomerTypeRequest(
        companySearch = null,
        entrepreneur = EntrepreneurCustomerInfo( <---------
            companyName = Property(companyName),
            ico = Property(ico.value),
            dic = Property(dic?.value)
        )
    )
}
```

**Comment by ursusursus** [comment]
works, thanks!
